### PR TITLE
oVirt template: send heartbeats to conformance lease

### PR DESCRIPTION
**Pull request by Gal-Zaidman**
- adds a command to send heartbeats to conformance leases,
- a few prints to get times for each lease

Signed-off-by: Gal Zaidman <gzaidman@redhat.com>

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 1]
```diff
@@ -417,6 +423,7 @@ objects:
 
         # What we're doing here is we generate manifests first and force that OpenShift SDN is configured.
         TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create manifests --log-level=debug
+        cat /tmp/artifacts/installer/manifests/cluster-infrastructure-02-config.yml
```
Comment (crawford) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: Is this needed?

Comment (rgolangh) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: That more for debugging purposes of CI failures. 
@Gal-Zaidman please leave a comment above this.

Comment (Gal-Zaidman) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: Added A comment

Comment (sdodson) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: This doesn't leak any form of long term credentials does it?

Comment (sdodson) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: It doesn't appear to from the rehearsal logs.

**Pull request merged**